uploader/src/main/resources/cfn-template.yaml

```diff
@@ -0,0 +1,139 @@
+AWSTemplateFormatVersion: "2010-09-09"
```

Comment:
We should document the cloudformation template for uploader is separate to the rest of the stack.